lib/removeUnusedVertices.js

```diff
@@ -132,7 +146,15 @@ function calculateIndexIntervals(accessors, buffers, bufferViews, meshes) {
                         var accessorIndex = 0;
                         var accessorSourceLength = accessorSource.length;
                         while (accessorIndex < accessorSourceLength) {
-                            var index = WebGLConstants.UNSIGNED_SHORT ? accessorSource.readUInt16LE(accessorIndex) : accessorSource.readUInt8LE(accessorIndex);
+                            var index = 0;
+                            switch(indexStride) {
+                                case 2:
+                                    index = accessorSource.readUInt16LE(accessorIndex);
+                                    break;
+                                case 1:
+                                    index = accessorSource[accessorIndex];
+                                    break;
+                            }

```

Comment:
Yeah don't worry about floats, if anything we may want to handle uint32, even though it's an extension.